src/genie/libs/parser/iosxe/show_cts.py

```diff
@@ -0,0 +1,119 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ======================
+# Schema for:
+#  * 'show cts rbacl'
+# ======================
+class ShowCtsRbaclSchema(MetaParser):
+    """Schema for show cts rbacl."""
+
+    schema = {
+        "cts_rbacl": {
+            "ip_ver_support": str,
+            str: {
+                "ip_protocol_version": str,
+                "refcnt": int,
+                "flag": str,
+                "stale": bool,
+                Optional(int): {
+                    Optional("action"): str,
+                    Optional("protocol"): str,
+                    Optional("direction"): str,
+                    Optional("port"): int
+                }
+            }
+        }
+    }
+
+
+# ======================
+# Parser for:
+#  * 'show cts rbacl'
+# ======================
+class ShowCtsRbacl(ShowCtsRbaclSchema):
+    """Parser for show cts rbacl"""
+
+    cli_command = ['show cts rbacl']
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command[0])
+        else:
+            out = output
+
+        cts_rbacl_dict = {}
+
+        ip_ver_capture = re.compile(r"^RBACL\s+IP\s+Version\s+Supported:\s(?P<ip_ver_support>.*$)")
+        rbacl_capture = re.compile(r"^(?P<rbacl_key>.*)(?==)=\s+(?P<rbacl_value>.*$)")
+        rbacl_ace_capture = re.compile(
+            r"^(?P<action>(permit|deny))\s+(?P<protocol>\S+)(\s+(?P<direction>dst|src)\s+((?P<port_condition>)\S+)\s+(?P<port>\d+)|)")
+
+        remove_lines = ('CTS RBACL Policy', '================', 'RBACL ACEs:')
+
+        # Remove unwanted lines from raw text
+        def filter_lines(raw_output, remove_lines):
+            # Remove empty lines
+            clean_lines = list(filter(None, raw_output.splitlines()))
+            rendered_lines = []
+            for clean_line in clean_lines:
+                clean_line_strip = clean_line.strip()
+                if not clean_line_strip.startswith(remove_lines):
+                    rendered_lines.append(clean_line_strip)
+            return rendered_lines
+
+        out = filter_lines(raw_output=out, remove_lines=remove_lines)
+        rbacl_name = ''
+        rbacl_ace_index = 1
+        for line in out:
+            ip_ver_match = ip_ver_capture.match(line)
+            if ip_ver_match:
```

Comment:
Instead of 
```
            ip_ver_match = ip_ver_capture.match(line)
            if ip_ver_match:
```
can you do? 
```
            if ip_ver_capture.match(line):
                ip_ver_match = ip_ver_capture.match(line)
```
This way you can convert the other if's to elif's? I think having the if, elif structure reaffirms the basic structure.

Reply:
Captures and if statements have been updated.